README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
+
+## Welcome! 👋
+> #### Cars BnB App
+> This is aone stop shop for reservation of exotic cars test driving.
+
+## Back-end App
+> [Back-end](https://github.com/Abdo9826/cars-bnb-backend)
+                             
+## Built With ✔️
+
+- React 🔥
+- redux toolkit
+- Bootstrap CSS 😋
+
+
+## Live link 🙌
+
+[LINK](https://6365ab83111cad2f422103df--stately-tulumba-fce256.netlify.app/)
+
+
+## Getting Started 🙌
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  node js
+  npm
+
+```
+### Setup
+Clone the repository and change the directory to portfolio
+
+``` 
+  `$ git clone https://github.com/Abdo9826/cars-bnb-frontend.git`
+
+  `$ cd cars-bnb-frontend`
+
+```
+
+### Install
+Install all project dependencies by running the command below
+ 
+``` 
+ ` $ npm install`
+```
+### Usage
+
+Run
+``` 
+  `$ npm start or deploy the index.html from the build/public folder 
+```
+
+### Deployment
+- [Visit Application Homepage](#)
+
+
+### Authors
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Robertson Akpan|[@bobb-rob](https://github.com/bobb-rob)|[@___Robertson](https://twitter.com/___Robertson)|[@RobertsonAkpan](https://www.linkedin.com/in/robertsonakpan/)|
+
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Abdo |[@abdo](https://github.com/Abdo9826)|[@Abdo](https://twitter.com/Abdo)|[@Abdo](https://www.linkedin.com/in/Abdo/)|
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Sigie |[@sigie](https://github.com/sigie)|[@Abdo](https://twitter.com/sigie)|[@sigie](https://www.linkedin.com/in/sigie/)|
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Shahnawaz Ali |[@Shahnawaz](https://github.com/Shahnawaz)|[@Abdo](https://twitter.com/Shahnawaz)|[@sigie](https://www.linkedin.com/in/Shahnawaz/)|
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to Microverse team for inspiring this project.
+
+## 📝 License
+
+All rights reserved.
```

Comment:
- [ ] On the "License" section of your readme, there is no link, and there is no license in your frontend project.

 ## Tips and extra help

Go [here] (https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪